lib/src/components/Tooltip/tooltip.module.scss

```diff
@@ -24,4 +23,9 @@
   .childItem {
     display: inline-block;
   }
+
+  /* Gap between tooltip elements should be: --components-dimensions-tooltip-gap-content */
+  .title {
```

Comment:
I think you can get rid of this code and the `.title` class and add the following to the root element:

```
display: flex;
flex-direction: column;
gap: var(--components-dimensions-tooltip-gap-content);
```